src/Microsoft.ML.Transforms/ProjectionCatalog.cs

```diff
@@ -7,6 +7,9 @@
 
 namespace Microsoft.ML
 {
+    /// <summary>
+    /// The catalog of projection transformations.
```

Comment:
>transformations [](start = 34, length = 15)

transformers?

Reply:
This is some troublesome language usage that we need to think about how to do. We are talking about transforming usage. And the way we do that is via `ITransformer`, but what is actually being returned through these catalogs? `IEstimator`! I don't object to this preference for transformer, per se, but I would not insist on it if it is unnatural in context. The context here is how we describe the process by which we *transform* data, which includes `ITransformer`, but is not specifically related to it (it can't be, what is returned here is not `ITransformer` implementors at all!).

Reply:
So I don't disagree with your suggestion, but I wouldn't insist on it, and I don't know that I see a strong reason for it, since the term usage seems appropriate in the (deliberately) vague context of this catalog.

Reply:
This is part of the `.Transforms` section of `MLContext`. I preferred not to use the contraction "transforms" and use the full word "transformations". My choice is more open ended and not reference ML.NET objects.

If we want to be more specific, since these are returning estimators, we should probably use that word instead.

So in conclusion, if we want to be specific I would use "estimators" alternatively I would stick to "transformations".  I would personally keep the word "transformations".